docs/dev/test-configuration-options.md

```diff
@@ -494,9 +517,16 @@ Allows you to set a path to an `.ipa`, `.apk`, `.aab` or `.zip` file containing
 
 <!-- prettier-ignore -->
 :::note
-- If this capability is not provided for a virtual device, the ['browserName'](#browserName) capability needs to be set. If none is set the test will throw an error.
-- This capability can be omitted for virtual devices if the ['browserName'](#browserName) capability is set.
-- If this capability is not provided for a real device session and also the ['browserName'](#browserName) capability is not provided then a real device session will automatically default back to the default browser. This will be Chrome for Android and Safari for iOS.
+
+- If this capability is not provided for a virtual device, the [`browserName`](#browsername-1) capability needs to be set. If none is set the test will throw an error.
+- This capability can be omitted for virtual devices if the [`browserName`](#browsername-1) capability is set.
+- If this capability is not provided for a real device session and also the:
+  - [`browserName`](#browsername-1) capability
+  - or [`appium:app`](#appiumapp)
+  - or [`appium:bundleId`](#appiumbundleid) (iOS)
+  - or [`appium:appPackage`](#appiumapppackage) and ['appium:appActivity'](#appiumappactivity) (Android)
+
+capability are not provided, then a real device session will automatically default back to the default browser. This will be Chrome for Android and Safari for iOS
```

Comment:
```suggestion
capability are not provided, then a real device session will automatically fallback to the default browser. This will be Chrome for Android and Safari for iOS
```